src/Infrastructure/LeanCode.ForceUpdate.Contracts/VersionSupport.cs

```diff
@@ -0,0 +1,16 @@
+using LeanCode.Contracts;
+
+namespace LeanCode.ForceUpdate.Contracts;
+
+public class VersionSupport : IQuery<VersionSupportDTO?>
+{
+    public PlatformDTO Platform { get; set; }
+    public string Version { get; set; } = default!;
```

Comment:
Imho `= default!;` in contracts is just noise, instead consider disabling nullable warnings in the entire `Contracts` project (assuming that `<Nullable>enable</Nullable>` in `Directory.Build.props` won't interfere :see_no_evil:).